js/ngDialog.js

```diff
@@ -26,6 +26,12 @@
 	var animationEndSupport = isDef(style.animation) || isDef(style.WebkitAnimation) || isDef(style.MozAnimation) || isDef(style.MsAnimation) || isDef(style.OAnimation);
 	var animationEndEvent = 'animationend webkitAnimationEnd mozAnimationEnd MSAnimationEnd oanimationend';
 	var focusableElementSelector = 'a[href], area[href], input:not([disabled]), select:not([disabled]), textarea:not([disabled]), button:not([disabled]), iframe, object, embed, *[tabindex], *[contenteditable]';
+	var reqAnimationFrame = window.requestAnimationFrame ||
```

Comment:
$window is not injected at this line. I can move the variable "reqAnimationFrame" into the $get method but i think it should be better to be declared after animationEndSupport, animationEndEvent, ...